content/technology/Material UI/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Material UI
+description: Material UI is one of the popular react UI framework used in Altinn Studio/ Altinn Studio Apps
+weight: 100
+tags: ["tjenester 3.0", "MAterial UI"]
+---
+
+#### Material UI 
+Material UI is used in Altinn Studio as the React UI framework. MAterial UI was chosen because
+- It is based on the Google's material design. 
+- It focuses exclusively on the React library
+- Can use the readily available responsive components
```

Comment:
Can use responsive components provided by the library

Reply:
Fixed